libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushService.java

```diff
@@ -119,22 +119,11 @@ static void runIntentInService(Intent intent) {
             WAKE_LOCK.acquire(WAKE_LOCK_TIMEOUT_IN_MILLIS);
         }
         intent.setClassName(context, SalesforceSDKManager.getInstance().getPushServiceType().getName());
-        final ComponentName name = context.startService(intent);
-        if (name == null) {
-            SalesforceSDKLogger.w(TAG, "Could not start GCM service");
-        }
+        enqueueWork(context, SalesforceSDKManager.getInstance().getPushServiceType(), JOB_ID, intent);
     }
 
-	/**
-	 * Default constructor.
-	 */
-	public PushService() {
-		super(TAG);
-		context = SalesforceSDKManager.getInstance().getAppContext();
```

Comment:
Why not keep that?

Reply:
Because there's no such constructor in the super class. It could be invoked statically.